### PR TITLE
2.41.2

### PR DESCRIPTION
**Pull request by catalinaadam**

### CODE REVIEW

**Review by rnovikovP — APPROVED** [review]
I've checked only Release highlights and they are 👌 from my side

File: docs/release-notes/2.41.2.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+![!image](../_images/patroni_dashboard.png)
+
+!!! caution alert alert-warning "Important"
+    We would appreciate your feedback on these experimental dashboards to help us enhance it further.
+
+
+## Improvements
+
+
+[PMM-10974](https://perconadev.atlassian.net/browse/PMM-10974) - By default, PMM used `pg_stat_statements` for **stat tracking**, but now it uses `pg_stat_monitor`, which provides additional metrics and supports **explains/examples** functionality that `pg_stat_statements` does not.
```
Comment (BupycHuk) [inline docs/release-notes/2.41.2.md:93]: it actually already used to be pg_stat_monitor as default for some time already. The change we included in 2.41.2 is that when pg_stat_monitor isn't available it fallbacks to pg_stat_statements.

File: docs/release-notes/2.41.2.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+[PMM-12884](https://perconadev.atlassian.net/browse/PMM-12884) - In PMM dump, the **Address** field on the **Send to Support** screen now comes with the address and port pre-filled, making it easier to send logs to support.
+
+[PMM-12887](https://perconadev.atlassian.net/browse/PMM-12887) - [Tech Preview] This new experimental dashboard has been added in PMM 2.41.2 for monitoring PostgreSQL checkpoints, buffers and WAL usage.
+
+[PMM-12888](https://perconadev.atlassian.net/browse/PMM-12888) - [Tech Preview] This new experimental dashboard has been added in PMM 2.41.2 to monitor [Patroni](https://patroni.readthedocs.io/en/latest/) as an external service.
+
+[PMM-12960](https://perconadev.atlassian.net/browse/PMM-12960) - [Tech Preview] The latest PMM release includes redesigned Kubernetes dashboards with improved clarity and usability.
+
+[PMM-12806](https://perconadev.atlassian.net/browse/PMM-12806)- In prior versions, PMM Server would prioritize certain VictoriaMetrics command line flags over environment variables. This behavior has now been addressed.
```
Comment (BupycHuk) [inline docs/release-notes/2.41.2.md:103]:
```suggestion
[PMM-12806](https://perconadev.atlassian.net/browse/PMM-12806)- In prior versions, PMM Server had certain non-configurable Victoria Metrics settings with default values. This behavior has now been addressed.
```

Comment (BupycHuk) [reply docs/release-notes/2.41.2.md:103]: and it's in both lists: improvements and bugs

File: docs/release-notes/2.41.2.md (hunk 1 above)
Comment (rasika-chivate) [reply docs/release-notes/2.41.2.md]: Fixed it. @BupycHuk please check

File: docs/release-notes/2.41.2.md (hunk 2 above)
Comment (rasika-chivate) [reply docs/release-notes/2.41.2.md:102]: Fixed. Added it in improvements.

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**